src/ralph/cli.py

```diff
@@ -164,6 +187,8 @@ def wrapper(command):
                         option_kwargs["type"] = CommaSeparatedKeyValueParamType()
                     elif field_type is CommaSeparatedTuple:
                         option_kwargs["type"] = CommaSeparatedTupleParamType()
+                    elif isclass(field_type) and issubclass(field_type, ClientOptions):
```

Comment:
`field_type` can be something else than a Class?

Reply:
Learned it the hard way, it can be a function (typing.Literal["index", "create", "delete", "update"] )